Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM node:lts-alpine AS builder
+
+# Working directory as specified by exercism
+WORKDIR /opt/analyzer
+
+# Copy source code
+COPY . .
```

Comment:
I would move this until after the curl commands, as the latter are less likely to change than the source code and thus docker layer caching can be used better.

Reply:
Oh, that's a clever advice, thank you!